client/src/components/NavBar/index.js

```diff
@@ -0,0 +1,137 @@
+import {
+  Avatar,
+  Box,
+  Button,
+  makeStyles,
+  Typography,
+  Menu,
+  MenuItem,
+} from "@material-ui/core";
+import { useContext, useState } from "react";
+import { NavLink } from "react-router-dom";
+import { UserContext } from "../../context/UserContext";
+import logo from "./ic-logo.png";
+
+const useStyles = makeStyles((theme) => ({
+  logo: {
+    height: 34,
+    width: 34,
+    marginRight: 8,
+  },
+  link: {
+    marginLeft: 50,
+    textDecoration: "none",
+    padding: 1,
+    color: theme.palette.common.black,
+  },
+  subLink: {
+    textDecoration: "none",
+    padding: 1,
+    color: theme.palette.common.black,
+  },
+  nav: {
+    display: "flex",
+    justifyContent: "space-between",
+    padding: 24,
```

Comment:
Would it be possible to change some of these values up to closer match the mock up? I tried adding 24px 36px here and adding margin of 18px by margin for the logo. It looked fairly similar to the mark up nav bar